packs/equipment/glorious-plate.json

```diff
@@ -61,7 +61,37 @@
         "resiliencyRune": {
             "value": "resilient"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.InCombat.ToggleLabel",
```

Comment:
Unfortunately using “encounter” as a predicate for TokenLight and TokenEffectIcon does not currently work, as it doesn't refresh when the state changes.